src/components/custom-field-input-date/__snapshots__/custom-field-input-date.test.jsx.snap

```diff
@@ -4,58 +4,54 @@ exports[`src/components/custom-field-input-date/custom-field-input-date classNam
 <body>
   <div>
     <div
-      class="container"
```

Comment:
In order to extract a "date control" to be shared between the field and cell components, I "simplified" the structure. Just moved some `<div>`s inside and moved the `<FormControl>` to the outside. We lost a little bit of padding but I think that's okay given the easy code re-use.